cumulusci/tasks/metadata_etl/value_sets.py

```diff
@@ -49,39 +44,27 @@ def _transform_entity(self, metadata, api_name):
                         "CaseStatus standard value set entries require the key 'closed'"
                     )
 
-            new_entry_index = get_new_tag_index(metadata, "standardValue")
+            metadata.append(tag="standardValue")
```

Comment:
I don't think this is right. We don't want to insert an empty `<standardValue>` tag yet.

Reply:
Good point. Fixed in af6c60eb